Rakefile

```diff
@@ -15,6 +15,20 @@ task :test_stable do
   abort 'Stable spec failed' unless RSpec::Core::Runner.run(['spec/cantaloupe_spec.rb']).zero?
 end
 
+desc 'Test the stable version with a DELEGATE_URL'
```

Comment:
We wouldn't need these with the matrix ENV option

Reply:
This is true for Travis, not true for running rake locally, unless I'm mistaken? Rake doesn't know about the Travis matrix, right?

Reply:
A Rakefile is just a Ruby script. The Travis matrices are just ENVs. If you put `puts ENV['FRANK']` in the rubocop task and then run `FRANK=bob rake rubocop` you'll see "bob" output. The only reason we put the CANTALOUPE_VERSION in the Rakefile is to simplify our running both of those two options for dev purposes. I think we could rely on Travis to run the delegate script tests (unless we wanted to run locally, then we could run with DELEGATE_URL=url-here rake). Setting the matrix wouldn't affect our local runs, that's right, though.

I was just proposing a simplification, not saying what you had wouldn't work or anything.